ros/src/computing/perception/localization/packages/orb_localizer/README

```diff
@@ -11,6 +11,18 @@ Requirements:
 - GLUT
 
 
+Notes for Large-scale Map
+ROSLaunch will raise signal SIGTERM and SIGKILL whenever a process timeouts
+after SIGINT. You may need to increase timeout values in the file
+/opt/ros/indigo/lib/python2.7/dist-packages/roslaunch/nodeprocess.py
+(change indigo to your ROS version).
+The default values are:
+_TIMEOUT_SIGINT  = 15.0 #seconds
+_TIMEOUT_SIGTERM = 2.0 #seconds
```

Comment:
this is not a very elegant solution to edit stuff in /opt/ros/indigo/lib/python2.7. Do you mean a SIGINT as a result of Ctrl+C or where does SIGINT come from?

Reply:
At this time I do not have good solution for this, because saving large ORB map eats too much time.